src/test/java/io/personium/core/auth/AccessContextTest.java

```diff
@@ -134,10 +152,9 @@ public void testGetRoleList() {
     }
 
     /**
-     * testCreateBasicのテスト.
      */
     @Test
-    public void testCreate() {
+    public void create_NoAuthzHeader_ShouldReturn_TypeAnonymous() {
```

Comment:
test name improvement.

```
{targetMethodName}_{Condition}_{Result}
```

I know in JUnit culture, people use the following method name convention.

```
test{targetMethodName}
```

The "test" part is obvious since it is a Test Class, and the we usually have input / result variations.